tests/test_git.py

```diff
@@ -584,6 +584,81 @@ def test_fetch_latest_items_from_empty_repository(self):
 
         shutil.rmtree(new_path)
 
+    def test_fetch_no_update(self):
+        """Test whether the fetch method properly works when the no update flag is set"""
```

Comment:
I would reword it like "Test whether new commits aren't fetched when no_update flag is set"